### PR TITLE
Add version on readme

### PR DESCRIPTION
**Pull request by kpman**

### CODE REVIEW

**Comment by tw0517tw** [comment]
should we say `Graph API` version?

**Comment by kpman** [comment]
Sure, I just added some related instructions on readme.

**Comment by tw0517tw** [comment]
What I mean is that `Facebook App version` is actually `Graph API version of the Facebook App`, should we mention that more clearly

**Comment by tw0517tw** [comment]
But in `bottender-fb` case, is that just setting the webhook subscription (of `feed`) version to 2.11 would be enough

File: README.md [hunk 1]
```diff
@@ -48,6 +57,26 @@ bot.onEvent(async context => {
...
+You can use other version by following this example:
+
+```js
+const { Bot } = require('bottender');
+const { FacebookConnector, FacebookClient } = require('bottender-fb');
+
+const bot = new Bot({
+  connector: new FacebookConnector({
+    appSecret: APP_SECRET,
+    client: new FacebookClient({
```
Comment (chentsulin) [inline README.md]: FacebookClient.connect?

**Pull request merged**